package.json

```diff
@@ -86,6 +86,7 @@
     "nopt": "^5.0.0",
     "npm-audit-report": "^2.1.4",
     "npm-package-arg": "^8.1.2",
+    "npm-packlist": "^2.2.2",
```

Comment:
Now that we have a proper release-next we should pull this in as an `npm update`, and then pull tap in like we normally do to update it.